presto-main/src/main/java/com/facebook/presto/sql/planner/PlanBuilder.java

```diff
@@ -90,6 +90,26 @@ public TranslationMap getTranslations()
         return translations;
     }
 
+    public PlanBuilder prependProjections(Iterable<Expression> expressions, SymbolAllocator symbolAllocator, PlanNodeIdAllocator idAllocator)
```

Comment:
It matters since `QualifiedNameReference` nodes are mapped to projection symbol via `ResolvedField.fieldIndex`